components/ai_chat/common/mojom/ai_chat.mojom

```diff
@@ -80,6 +82,10 @@ struct Model {
   // user-facing category
   ModelCategory category;
   bool is_premium;
+  // max limit to truncate page contents
+  uint32 max_page_content_length;
+  // max limit for overall conversation
+  uint32 max_conversation_length;
```

Comment:
I think this name needs adjustment. `max_conversation_length` implies that we should stop the conversation once it gets to be that length. But, this is the conversation character length at which we warn about a long conversation and that it might not be as accurate (and approaching the hard limit). It's not a hard limit itself. So perhaps the name could reflect that, e.g. `long_conversation_warning_character_limit`?

Reply:
changed